src/tests/test_strong_type.cpp

```diff
@@ -157,8 +158,7 @@ std::vector<Test::Result> test_container_strong_type() {
 
                Test_Length_List hashes({Test_Length(3), Test_Length(1), Test_Length(4), Test_Length(2)});
 
-               // TODO: C++20 - std::ranges::sort
-               std::sort(hashes.begin(), hashes.end());
+               std::ranges::sort(hashes);
```

Comment:
So far I didn't use the ranges algorithms because many aspects of the ranges library didn't work (well) with some older compiler we still have to support. Perhaps the situation is better now, or maybe `std::ranges::sort` doesn't fall in this category. If CI is green, I guess its fine. 😏